src/views/Join.vue

```diff
@@ -0,0 +1,259 @@
+<template>
+<div class = "page">
+
+    <div class = "header-container">
+        <div class = "header-group">
+            <h1 class = "header-title">Change the world and build projects with us.</h1>
+            <div class = "cta-button"> 
+                <h3> APPLY </h3>
+                <div id = "button-arrow">  </div>
+            </div>
+        </div>
+    </div>
+
+    <div class = "container-right">
+        <div class = "cr-image">
+         <img id="code-img" src="@/assets/img/code@2x.png" > 
```

Comment:
you may want to fix this indentation. Also, it is nice to always have a closing tag with an opening tag. however, there is a shorthand if nothing is there in between: `<img></img>` === `<img />`